### ISSUE TITLE
network doesn't reconnect

### ISSUE THREAD

**Issue by Tcll** [lbryio/lbry-desktop#4617]
I have to use a VPN to get around the desktop app crashing due to AWS being blocked by my firewall for being a malicious service...

but occasionally my VPN will stop serving traffic and I'll need to either switch servers or disconnect
in both cases I'll need to restart LBRY because it stops loading content
it's actually currently doing this as I type this message, comments and videos just won't load on anything.

**Comment by tzarebczan** [comment]
Sorry to hear about the trouble and thanks for the issue. This would be handled on the lbry-sdk side and there's an open issue for it: https://github.com/lbryio/lbry-sdk/issues/1277

**Comment by Tcll** [comment]
thanks for the reference
and yes just to confirm, this is not a new issue
I've actually been putting up with it through quite a few LBRY versions now and have been procrastinating about creating an issue.

I'm only doing so now as LBRY is incredibly painful to use on an Atom D2550 with 2GB of DDR3
